tools/generate-classes/glTF.json

```diff
@@ -162,6 +162,12 @@
     "KHR_draco_mesh_compression extension": {
       "overrideName": "ExtensionKhrDracoMeshCompression"
     },
+    "KHR_materials_variants glTF extension": {
+      "overrideName": "ExtensionKhrMaterialsVariants"
+    },
+    "KHR_materials_variants mesh primitive extension": {
+      "overrideName": "ExtensionKhrMaterialsVariantsMeshPrimitive"
+    },
```

Comment:
If an extension has multiple attachment points like `KHR_materials_variants` does, the naming convention is `Extension${AttachmentName}${ExtensionName}` (see `EXT_structural_metadata` for example):

* `ExtensionKhrMaterialsVariants` should be `ExtensionModelKhrMaterialsVariants`
* `ExtensionKhrMaterialsVariantsMeshPrimitive` should be `ExtensionMeshPrimitiveKhrMaterialsVariants`.

Reply:
I should have noticed that. The code (and the example code) have been updated.